src/main/java/org/dwcj/controls/DateEditBox.java

```diff
@@ -34,11 +40,21 @@ protected void create(AbstractDwcjPanel p) {
         }
     }
 
+    /**
+     * This method pops up a calendar dialog attached to the DateEditBox control
+     * 
+     */
     public void calendar() {
         ((BBjInputD) this.ctrl).calendar();
     }
 
-    public boolean getBeep(){
+    /**
+     * This method returns whether the DateEditBox control beeps on invalid input.
+     * 
+     * @returns Returns whether the control will beep on invalid input (false = No Beep, true = Beep).
+     * 
+     */
+    public boolean isBeep(){
```

Comment:
have we ever tried "beep" in DWC? If beep does not work, we ought to remove it...

Reply:
I haven't, and I'll give this a try before I include it.